### DISCUSSION TITLE
EV Charger SCharger-22KT-S0

### DISCUSSION THREAD

**Discussion by PStenbumling** [wlcrs/huawei_solar#358] [General]
Now that the Ev charger is out (SCharger-22KT-S0) any plans to support that? 
https://support.huawei.com/enterprise/en/doc/EDOC1100260967

**Comment by wlcrs** [comment]
I won't implement support for devices that I cannot test against. That would be a nightmare for both me and all the frustrated users because of all the bugs that I would not be able to catch with said testing.

Furthermore I doubt that it would be a good fit for this integration. A separate integration would make more sense. Not in the least as I expect that a separate connection to the charger unit will have to be made. The underlying library now assumes that there is only 1 connection to 1 device (which can have multiple subdevices like a power meter, battery or cascaded inverters)

**Reply by sphings79** [reply]
Hi, my SCharger is directly connected to the EMMA. So no way to get data on an other way. Would be nice if it would be possible to get the actual charging power?
Isn't EMMA able to deliver that?
